### DISCUSSION TITLE
Superset Period-over-Period Charts

### DISCUSSION THREAD

**Discussion by maki-mo** [apache/superset#16506] [Q&A / Help]
Is it possible to create Period-over-period charts in Superset so I can see data for a specified date range compared with data for the same range for another period (year on year, week on week, month on month...)? For example, you might have a period-over-period chart showing your earnings for the previous year side-by-side with the same measurements for earlier years.

<img width="643" alt="period over period" src="https://user-images.githubusercontent.com/89787958/131348982-455382bc-39b1-45cc-b7f4-e8fca06f6571.PNG">

**Comment by kamalkeshavani-aiinside** [comment]
Check 'Time Comparison' section under Advanced Analytics of any time-series chart.

**Reply by maki-mo** [reply]
Hi Kamal!
In fact I did, and I know about this feature, but unfortunately, for some reason the bars for both periods that are compared are all in the same colors so it doesn't help much.
Regards,
Marko

**Reply by kamalkeshavani-aiinside** [reply]
Which version of Superset are you using? In v1.3.0, I see this issue fixed.

**Reply by Baryk007** [reply]
Hello, 
how did you solve this issue? Facing the same issue with version 3.0.2
Thx

**Reply by Quatters** [reply]
Looks like after 3.0.0 (PR: https://github.com/apache/superset/pull/24048) child lines always have the same color as original one.
